tendenci/apps/events/tests.py

```diff
@@ -104,3 +106,19 @@
 #         with self.assertRaises(Addon.DoesNotExist):
 #             Addon.objects.get(title='Test Addon')
 #         logger.info('Complete.')
+
+class EventTest(TestCase):
+    def test_repeat_of_relationship(self):
+        original_event = baker.make('events.Event')
+        user = baker.make('auth.User')
+        # Add new event by setting 'repeat_of'
+        repeat_event = copy_event(original_event, user, set_repeat_of=True)
+        self.assertEqual(original_event.repeat_uuid, repeat_event.repeat_uuid)
```

Comment:
The first part of this text, adds a new Event & uses 'repeat_of' We should end up with the same repeat_uuid on both the original & the new, repeated event